spec/unit/puppetlabs_spec_helper/tasks/module_inspect_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe 'rake module:inspect', type: :task, unless: puppet_3_or_older? do
+  let(:manifest_dir) { File.join(Dir.pwd, 'manifests') }
+  let(:no_results) { { 'host_class' => [], 'defined_type' => [] } }
+
+  before(:each) do
+    Dir.mkdir(manifest_dir)
```

Comment:
Should this be cleaned up afterwards?

Reply:
Nope, these tests use FakeFS (automatically included in `:type => :task` specs) which is an in memory filesystem

Reply:
Ahh ok... hard to see that :-(  apologies